.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+    - "4.1"
+
+script: gulp test
```

Comment:
Surprised that this didn't need compile… I guess the prepublish script runs on install?